integration_test/test_videoroom/test/integration/stampede_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule TestVideoroom.Integration.ClientTest do
-  use TestVideoroomWeb.ConnCase, async: true
+  use TestVideoroomWeb.ConnCase, async: false
```

Comment:
If we would change async to true, the tests from different modules will run at the same time. Currently, the test app has only one room, so all stampedes would join the same room and this would lead to strange behavior when we expect in a test that we have 4 peers in the room, but we have 6 because an additional two are from the second module test.